package.json

```diff
@@ -6,18 +6,22 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.32",
     "@fortawesome/free-solid-svg-icons": "^5.15.1",
     "@fortawesome/react-fontawesome": "^0.1.11",
+    "@material-ui/core": "^4.11.0",
+    "@material-ui/lab": "^4.0.0-alpha.56",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
     "autoprefixer": "^8.0.0",
     "postcss-cli": "^7.1.0",
     "react": "^16.13.1",
+    "react-bootstrap": "^1.3.0",
     "react-digraph": "^7.0.0",
     "react-dom": "^16.13.1",
     "react-particles-js": "^3.3.0",
     "react-router-dom": "^5.2.0",
     "react-scripts": "3.4.3",
-    "tailwindcss": "^1.8.12"
+    "tailwindcss": "^1.8.12",
```

Comment:
Get some tailwind warnings when the consol is opened

Reply:
try running npm run build:css